joeynmt/prediction.py

```diff
@@ -475,10 +471,52 @@ def _translate_data(test_data):
 
     if use_cuda:
         model.to(device)
+        
+    params_dict['model'] = model
+    return params_dict
+    
+
+    
+    
+def translate(params,data,type_:int,output_path=None) -> None:
```

Comment:
`type_` as a variable name and as int type is not very expressive: it's not clear from the documentation of the function what it means. 
Please add docstrings to the function.